src/plugins/vis_types/vislib/public/vislib/percentage_mode_transform.ts

```diff
@@ -6,8 +6,6 @@
  * Side Public License, v 1.
  */
 
-// @ts-ignore
-import numeral from '@elastic/numeral';
```

Comment:
@VladLasitsa it looks like you were trying to have typescript ignore this unused import. Is it okay to remove it?

Reply:
I assume this module didn't have types when it was first imported, and the `// @ts-ignore` hid the fact that it was unused

Reply:
Looks like I've added this with ts-ignore but after that remove usage and completely forgot about import. Sorry about that.

Reply:
No worries, the machines will take care of this now